terasoluna-gfw-parent/pom.xml

```diff
@@ -597,21 +607,27 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-gpg-plugin</artifactId>
-            <version>${maven-gpg-plugin.version}</version>
+            <version>${org.apache.maven.plugins.maven-gpg-plugin.version}</version>
             <executions>
               <execution>
                 <id>sign-artifacts</id>
                 <phase>verify</phase>
                 <goals>
                   <goal>sign</goal>
                 </goals>
+                <configuration>
+                  <gpgArguments>
+                    <arg>--pinentry-mode</arg>
+                    <arg>loopback</arg>
+                  </gpgArguments>
+                </configuration>
```

Comment:
- [ ] There is a user impact, so let's consider it a little more.

Reply:
The central profile set in terasoluna-gfw-parent is set to deploy parent, common-libraries, depenedensies to Maven central.
Therefore, it is unlikely that it will be used by the user. Also, the migration guide for version 5.2.0, which implements the central profile, does not mention that.
However, these settings are not always used by the user, so it is necessary to consider the contents described in the migration guide.

Reply:
It is OK because there seems to be no problem even if you set it here.
However, I wonder if it can be handled by commands etc. separately, but is it impossible?

Reply:
Since User Property is not set in "gpgArguments", it is not possible to input with a command, and it seems that there is no choice but to describe it in pom.
https://maven.apache.org/plugins/maven-gpg-plugin/sign-mojo.html#gpgArguments